src/main/java/openfasttrack/cli/CommandLineInterpreter.java

```diff
@@ -116,10 +121,30 @@ public void parse()
         }
     }
 
+    private void handleSingleCharacterArgument(final ListIterator<String> iterator,
+            final String argument)
+    {
+        final String arguments = argument.replaceFirst(SINGLE_CHAR_ARG_PREFIX, "").toLowerCase();
+
+        int position = 0;
+        for (final String character : arguments.split(""))
```

Comment:
Rewrote with regular counting loop. Used "substring" since I need Strings not chars to reuse existing functions.